### DISCUSSION TITLE
502 Gateway Error

### DISCUSSION THREAD

**Discussion by Haseeb-Usmani** [community/community#40014] [GitHub Education]
while setting up my GITHUB STUDENT PACK i have faced an issue, after i upload my student proof i clicked on submit button but the request is not proceeded so i reloaded the page and then 502 GATEWAY error occured.

**Comment by kingfisherDS** [comment]
I experience the same problem at the moment. Please let me know if you found a fix. Probably its temporary for the time it needs to verify the student status.

**Comment by pythonreactor** [comment]
Still an issue

**Comment by burkeac** [comment]
Was having this issue as well. Tried GitHub support, but they just closed the ticket and directed here.
